### PR TITLE
 numberFormat functoin

### PR DESCRIPTION
**Pull request by clever-cr**

### CODE REVIEW

**Review by haroldmud — APPROVED** [review]

File: numberFormat.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+var numberFormat = function (number) {
+    return number.toLocaleString()
+};
```
Comment (IbrahimBagalwa) [inline numberFormat.js]: You can improve this solution with ES6
```js
const numberFormat = (number) => number.toLocaleString()
```

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
good job @clever-cr 
i left a comment to improve the code

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
good job @clever-cr 
i left a comment to improve the code

**Review by IbrahimBagalwa — APPROVED** [review]
LGTM 👍

**Pull request merged**